### PR TITLE
use express library to print all the names except my name

### PR DESCRIPTION
**Pull request by chocobrownie**

### CODE REVIEW

File: expressPrintNotMe/app.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+const e = require("express");
+const app = e();
+const f = require("fs");
+app.listen(3611);
+app.get("/",(req,res) => {
+  let namesNotMe = [];
+  f.readdir("/home",
+    (err,files)=>{
```
Comment (allopez7) [inline expressPrintNotMe/app.js]: The code looks good!

**Review by allopez7 — APPROVED** [review]

File: expressPrintNotMe/app.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+const e = require("express");
+const app = e();
+const f = require("fs");
+app.listen(3611);
+app.get("/",(req,res) => {
+  let namesNotMe = [];
+  f.readdir("/home",
```
Comment (songz) [inline expressPrintNotMe/app.js]: `f.readdir('/home', (err, files) => { ... `

Comment (songz) [reply expressPrintNotMe/app.js]: don't break into multiple lines unless one line is getting too long

File: expressPrintNotMe/app.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    (err,files)=>{
+      files.forEach(oneFile=>{
+        if (oneFile != "ming") {
+          namesNotMe.push(oneFile);
+        }
+      });
+  res.send(namesNotMe);
+    });
```
Comment (songz) [inline expressPrintNotMe/app.js]: something looks really weird with indentation.

**Review by songz — APPROVED** [review]

**Pull request merged**